recipes/dash-cytoscape/meta.yaml

```diff
@@ -1,40 +0,0 @@
-{% set name = "dash_cytoscape" %}
-{% set version = "0.1.1" %}
-
```

Comment:
Please do not remove this package ...

Reply:
Thanks for catching this. I had an error when pushing to my staged-recipes. Fixed.